org.freedesktop.tuhi.json

```diff
@@ -127,7 +127,7 @@
                 "python3 setup.py install --verbose"
             ],
             "post-install": [
-                "cp tools/tuhi-kete.py /app/lib/python3.6/site-packages/tuhi_kete.py",
+                "cp tools/tuhi_kete.py /app/lib/python3.6/site-packages/tuhi_kete.py",
```

Comment:
Any reason we can't just name this thing `kete.py` and only name it `tuhi-kete`  on install? Seems better than messing around with underscores.

Reply:
OK, I'll amend this in #70